### DISCUSSION TITLE
Installing Congo Example

### DISCUSSION THREAD

**Discussion by xrayleak** [jpanther/congo#130] [Q&A]
I am very new to this and am having trouble getting the Congo Example to work. I noticed that within the congo-dev folder there are files like tailwind.config and lighthouserc. Do I need to place these in my project folder?

I started with a blank project that worked, then added in config, content and layout files from the example and get this error:

C:\Hugo\xrayleakWebsite>hugo server
Start building sites …
hugo v0.92.2-CDF6A0D6 windows/amd64 BuildDate=2022-02-11T14:17:39Z VendorInfo=gohugoio
Error: Error building site: "C:\Hugo\xrayleakWebsite\content\_index.md:6:1": failed to extract shortcode: template for shortcode "lead" not found
Built in 0 ms

Thank you, 

Brent

**Comment by OCram85** [comment]
Just updated my blog to congo 2 an get the same error...

**Comment by OCram85** [comment]
Just add this to your `config.toml`

```
theme= "congo"
```

In the documents of congo 1 there was an instruction how to set up hugo + congo with git based themes. This included an instruction to add the line in the `config.toml`. But it's missing in the new docs and *How to upgrade* docs....

**Reply by cbelena** [reply]
Hi, [that it is still detailed in the docs](https://jpanther.github.io/congo/docs/installation/#set-up-theme-configuration-files). I think that is not in the new docs about v2 because that didn't change between v1 and v2.
